public/bitacora.html

```diff
@@ -1,122 +1,150 @@
+                <img src="img/search-interface-symbol.png" alt="user-icon" />
+            </div>
+            <button type="button" class="buttons" onclick="Logout()">
+                Cerrar Sesión
+            </button>
         </div>
-        <button type="button" class="buttons" onclick="Logout()">
-          Cerrar Sesión
-        </button>
-      </div>
     </header>
 
     <main>
-      <h2 class="subtitles">Bitacora</h2>
-      <h2 class="subtitles">Reporte de activos</h2>
-      <p>SICA - FOLIO: 001</p>
-      <div class="container">
-        <div class="inputs">
-          <label for="nombre-empleado">Nombre del empleado</label>
-          <input type="text" id="nombre-empleado" />
+        <div id="miModal" class="modal">
+            <div class="modalContent">
+            <span class="close">&times;</span>
+        <form action="">
+            <div class="column">
+                <div class="titulo">
+                    <h2>Datos de Traslado</h2>
+                </div>
+                <div class="inputLabel">
+                    <div class="empleado">
+                        <label for="txtEmpleado" id="lblNombre">Nombre Del Empleado</label>
+                        <input type="text" id="txtEmpleado"> <br>
+                    </div>
+                    <div class="localizacion">
+                        <label for="txtLocalizacion" id="lblLocalizacion">Localización</label>
+                        <input type="text" id="txtLocalizacion"><br>
+                    </div>
+                    <div class="activo">
+                        <label for="txtActivo" id="lblActivo">ID de Activo</label>
+                        <input type="text" id="txtActivo"><br>
+                    </div>
+                    <div class="fecha">
+                        <label for="txtFecha" id="lblTraslado">Fecha del traslado</label>
+                        <input type="date" id="txtFecha"><br>
+                    </div>
+                    <div class="respon">
+                        <label for="txtRespon" id="lblRespon">Responsable de aceptar o denegar el traslado</label>
+                        <input type="text" id="txtRespon"><br>
+                    </div>
+                </div>
+            </div>
+            <div class="titulo">
+                <h2>Extras del traslado</h2>
+            </div>
+            <div class="inputLabel">
+                <div class="extra">
+                    <label for="txtExtra" id="lblExtras">Extras del traslado</label>
+                    <input type="text" id="txtExtra"><br>
+                </div>
+                <div class="donalmacen">
+                    <label for="txtelec" id="lblElec">Sera almacenado o donado</label>
+                        <select name="" id="txtEstado">
+                            <option value="1" checked>Donación</option>
+                            <option value="0">almacenado</option>
+                        </select>
```

Comment:
```suggestion
                        <select name="" id="txtEstado">
                            <option value="">--Seleccione una opción--</option>
                            <option value="1">Donación</option>
                            <option value="2">Almacenado</option>
                        </select>
```
- Hay que corregir los valores. Debe haber uno con valor vacío (el primero que siempre va a estar por defecto seleccionado) para que se hagan más sencillas las validaciones